src/Sarif.Viewer.VisualStudio/CloseSarifLogService.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.Sarif.Converters;
+using Microsoft.Sarif.Viewer.ErrorList; 
+
+namespace Microsoft.Sarif.Viewer
+{
+    public class CloseSarifLogService : SCloseSarifLogService, ICloseSarifLogService
+    {
+        /// <summary>
+        /// Closes all SARIF logs opened in the viewer.
+        /// </summary>
+        public void CloseAllSarifLogs()
+        {
+            ErrorListService.CloseAllSarifLogs();
+        }
+
+        /// <summary>
+        /// Closes the specified SARIF log in the viewer.
+        /// </summary>
+        /// <param name="paths">The complete path to the SARIF log file.</param>
+        public void CloseSarifLogs(IEnumerable<string> paths)
+        {
+            ErrorListService.CloseSarifLogs(paths);
+        }
+
+        public void LoadSarifLog(string path)
```

Comment:
I don't understand why this is here on a Close object.  Also is this not part of ILoadSarifLogService?

Reply:
Copy paste error. Removed

---
In reply to: [440935382](https://github.com/microsoft/sarif-visualstudio-extension/pull/143#discussion_r440935382) [](ancestors = 440935382)